portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -14,19 +14,36 @@
 
 package com.google.sps.servlets;
 
+import com.google.gson.Gson;
 import java.io.IOException;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
+import java.util.Arrays;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  private ArrayList<String> commentsList = new ArrayList<>();
```

Comment:
Use just List<String> as the declared type here. 
Why we try to declare types that are as abstract as possible? - Its so that we have the flexibility to use different implementations of the type without breaking anything when possible - so in this case, if we're not using anything ArrayList specific, it should just be 
private List<String> commentsList = new ArrayList<>();

Reply:
got it. pretty sure I caught this one in a different branch

Reply:
List would still be generically typed though right? `List<String> commentsList`

Reply:
yeah it'll be List<String>